src/field/scrcmd_pokemon_misc.c

```diff
@@ -41,6 +41,8 @@
 #include "unk_02037C94.h"
 #include "sound_02004A44.h"
 #include "constants/map_sections.h"
+#include "msgdata/msg.naix"
+// #include "application/zukanlist/zukan_data/zukan_data.naix"
```

Comment:
Any reason why this is commented out and not just removed?

Reply:
Because the naix does not exist yet.